app/assets/config/manifest.js

```diff
@@ -1,3 +1,4 @@
 //= link_tree ../images
 //= link_directory ../javascripts .js
 //= link_directory ../stylesheets .css
+//= link application.css
```

Comment:
@rsanheim strangely, my server yelled if I didn't add this here. Lmk if I should remove for merge.